rmgpy/rmg/main.py

```diff
@@ -228,13 +231,13 @@ def clear(self):
         self.uncertainty = None
 
         self.execTime = []
-    
+
     def loadInput(self, path=None):
         """
         Load an RMG job from the input file located at `inputFile`, or
         from the `inputFile` attribute if not given as a parameter.
         """
-        from input import readInputFile
+        from .input import readInputFile
```

Comment:
Replace with absolute import.

Reply:
Thanks for the catch! I forgot to check import statements nested in the code. I did some further searching using regex in PyCharm and I didn't find any further instances in the files covered by this PR, so I think we have it all now.